session/session_test.go

```diff
@@ -2018,7 +2018,7 @@ func (s *testSchemaSuite) TestTableReaderChunk(c *C) {
 	}
 	c.Assert(count, Equals, 100)
 	// FIXME: revert this result to new group value after distsql can handle initChunkSize.
-	c.Assert(numChunks, Equals, 1)
+	c.Assert(numChunks, Equals, 10)
```

Comment:
Why it's changed to `10`?

Reply:
In the test case, it `SplitTable` to 10 regions and will get 10 chunks in response.
In default format, it will append all result to one chunk.
In arrow format, it will return the chunk directly. So `numChunks` is changed to 10.